src/Components/Publishing/Byline/__tests__/__snapshots__/ShareDate.test.tsx.snap

```diff
@@ -50,7 +50,7 @@ exports[`renders ShareDate properly 1`] = `
   color: black;
 }
 
-@media (max-width:768px) {
+@media (max-width:768pxpx) {
```

Comment:
Looks like a typo?

Reply:
Good catch — probably some theme that didn't get upgraded back when. Will investigate.